openhtf/io/frontend/templates/station.html

```diff
@@ -30,7 +30,7 @@
             document.getElementById(id).innerHTML = new_doc; });
   }
   function poll(interval_ms) {
```

Comment:
This was hastily-produced code and my bad. I'm actually halfway through overhauling it and the next iteration should look a lot better. I'll include it in this PR when it's ready.